mol/molecule.py

```diff
@@ -3266,6 +3266,20 @@ def readrkf(self, filename: str_type, section: str_type = "Molecule", **other):
         for bo in self.bonds:
             bo.mol = self
 
+    def readcoskf(self, filename: str, **other):
```

Comment:
Please update the `Added` section of: https://github.com/SCM-NV/PLAMS/blob/trunk/CHANGELOG.md